os/hal/ports/STM32/LLD/FDCANv1/hal_can_lld.c

```diff
@@ -276,15 +276,22 @@ bool can_lld_start(CANDriver *canp) {
   }
 
   /* Configuration can be performed now.*/
-  canp->fdcan->CCCR   = FDCAN_CCCR_CCE;
+  canp->fdcan->CCCR  |= FDCAN_CCCR_CCE;
+  canp->fdcan->CCCR &= ~(FDCAN_CCCR_CSR | FDCAN_CCCR_CSA);

```

Comment:
another bug - this meant that any bits passed in to the TEST or CCCR registers didn't get applied because this line was previously clearing the INIT flag that allows writing those registers.